static/panes/ast-view.ts

```diff
@@ -0,0 +1,284 @@
+// Copyright (c) 2017, Simon Brand
```

Comment:
Quick question: Should this line be updated or left with the original copyright from ast-view.js?

Reply:
I think it should be left alone. We can ask @TartanLlama if they'd like us to update the attribution, but for best I'd leave the copyright alone wtihout clear intent from the original author. Thanks!